src/index.html

```diff
@@ -19,7 +19,7 @@ <h1 class="mb-0 h5 mr-auto py-1">Bootstrap Customizer</h1>
       <div class="card bg-light text-center">
         <div class="card-body">
           <h5 class="card-title">Customize and download</h5>
-          <p class="card-text">Create a custom <a href="https://getbootstrap.com/" title="Bootstrap website">Bootstrap</a> v4.x build by choosing the components and JavaScript plugins you need.</p>
+          <p class="card-text">Create a custom <a href="https://getbootstrap.com/" title="Bootstrap website">Bootstrap</a> v<span id="bsVersion"></span> build by choosing the components and JavaScript plugins you need.</p>
```

Comment:
Don't really like this. Maybe there is a way to render the number while building?

Reply:
yep, you're right, I'll do that with Webpack !